tools/cli/help.txt

```diff
@@ -534,14 +534,14 @@ Options:
                     test app rebuild.
 >>> test-app
 Test the application
-Usage: meteor test-app [options]
-       meteor test-app --unit [options]
-       meteor test-app --integration [options]
+Usage: meteor test-app --driver-package <driver> [options]
+       meteor test-app --driver-package <driver> --unit [options]
+       meteor test-app --driver-package <driver> --integration [options]
        
-Runs tests against the application, either in *unit* or *integration* 
-test mode. meteor test-app will start a special app based on a test driver
-which handles the task of running tests and displaying the results in the 
-browser when you visit it.
+Runs tests against the application, either in *unit* or *integration*
+(the default) test mode. meteor test-app will start a special app based on
+a test driver (specified with --driver-package) which handles the task of
+running tests and displaying the results in the browser when you visit it.
 
```

Comment:
I wonder now, how will people find which driver packages they should use? Should this help article link to the guide where we describe this?